src/App.js

```diff
@@ -13,15 +13,15 @@ function App() {
 
     const [pokemonCardData, setPokemonCardData] = useState({
         name: "",
-        pokemonType: "",
+        pokemonType: " ",
```

Comment:
you can remove this now, we no longer need it as we have added type to replace pokemonType
Also you can probably remove all the default data in here and just make it an empty object